src/cls.ts

```diff
@@ -21,6 +21,15 @@ import {SpanDataType} from './constants';
 import {UNCORRELATED_SPAN, UNTRACED_SPAN} from './span-data';
 import {Trace, TraceSpan} from './trace';
 
+export type RealRootContext = {
+  readonly span: TraceSpan; readonly trace: Trace; readonly type: SpanDataType
```

Comment:
nit: Do these need to be on the same line?

Reply:
`clang-format` says: Yep

Reply:
You should prefer `interface` over `type`. clang-format will format it more beautifully.

Reply:
You're right. Updated